docker/wordpress/wp-content/plugins/a/composer.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "jetpack/a",
+  "description": "a",
+  "type": "wordpress-plugin",
+  "require": {
+    "automattic/jetpack-logo": "1.0.0"
+  },
+  "require-dev": {
+    "automattic/jetpack-autoloader": "1.0.0"
+  },
+  "repositories": [
```

Comment:
We can totally remove the `logo` entry from the repositories now, since we released the Packagist package.